### PR TITLE
Schedule and Responsibilities

### PR DESCRIPTION
**Pull request by mitas1**
This PR contains: 
- Makefile for generating a specification
- spliting chapters to different files 
- new chapter Schedule and team responsibilities

### CODE REVIEW

**Comment by mskrip** [comment]
LGTM

File: docs/specification/06.tex [hunk 1]
```diff
@@ -0,0 +1,36 @@
+\chapter{Schedule and responsibilites}
+
+In this chapter we provide the schedle of development and information about
```
Comment (mskrip) [inline docs/specification/06.tex]: typo

File: docs/specification/06.tex [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+\end{itemize}
+
+\section{Team members and responsibilities}
+
+Our team consists of three members with following responsibilities:
+
+\begin{itemize}
+\item Marián Skrip - backend development and the head of our team
+\item Tomáš Sláma - frontend development, designing visuals and page layouts
+\item Samuel Mitas - backend development
```
Comment (mskrip) [inline docs/specification/06.tex]: I think you may have something to say in frontend dev as well

File: docs/specification/06.tex [hunk 3]
```diff
@@ -0,0 +1,36 @@
+\chapter{Schedule and responsibilites}
+
+In this chapter we provide the schedle of development and information about
+responsibilities in our team.
+
+\section{Schedule}
+
+\begin{itemize}
+\item Specification - 30.october 2017 - 6 man-days
+\item website design, visuals - 8.november 2017 -  2 man-days
```
Comment (mskrip) [inline docs/specification/06.tex]: Capitalize please

**Comment by mitas1** [comment]
Changed as requested. Take a look again, pls.

**Comment by mskrip** [comment]
LGTM

**Pull request merged**